TLPhotoPicker/Classes/TLPhotosPickerViewController.swift

```diff
@@ -379,12 +379,17 @@ extension TLPhotosPickerViewController {
     
     private func centerAtRect(image: UIImage?, rect: CGRect, bgColor: UIColor = UIColor.white) -> UIImage? {
         guard let image = image else { return nil }
-        UIGraphicsBeginImageContextWithOptions(rect.size, false, image.scale)
-        bgColor.setFill()
-        UIRectFill(CGRect(x: 0, y: 0, width: rect.size.width, height: rect.size.height))
-        image.draw(in: CGRect(x:rect.size.width/2 - image.size.width/2, y:rect.size.height/2 - image.size.height/2, width:image.size.width, height:image.size.height))
-        let result = UIGraphicsGetImageFromCurrentImageContext()
-        UIGraphicsEndImageContext()
+        
+        var format = UIGraphicsImageRendererFormat()
```

Comment:
This is also used in uploading a photo for nutrition events, looks like this functionality still works